spring-cloud-kubernetes-leader/src/main/java/org/springframework/cloud/kubernetes/leader/LeaderProperties.java

```diff
@@ -61,12 +58,6 @@
 	 */
 	private String configMapName = DEFAULT_CONFIG_MAP_NAME;
 
-	/**
-	 * Kubernetes labels common to all leadership candidates.
-	 * Default: empty
-	 */
-	private Map<String, String> labels = Collections.emptyMap();
-
```

Comment:
Can you add enabled property that is checked in LeaderAutoConfiguration?

Reply:
It is already here https://github.com/spring-cloud-incubator/spring-cloud-kubernetes/blob/master/spring-cloud-kubernetes-leader/src/main/java/org/springframework/cloud/kubernetes/leader/LeaderProperties.java#L46

Reply:
You're referring to autoStartup. I'm referring to "spring.cloud.kubernetes.leader.enabled", which is missing from LeaderProperties but checked in this line `@ConditionalOnProperty(value = "spring.cloud.kubernetes.leader.enabled", matchIfMissing = true)`

Reply:
Done. Although, it's not strictly necessary.